ops/ccs-ops-misc/synthetic-data/scripts/test_synthetic_bene_file.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+def validate_file(filename):
+    '''Validates a beneficiary csv and ensures that the file is valid for loading
+    into the pipeline. This is primarily designed to check synthea files for minor
+    errors post-generation. Validations can be added here as needed.
+    '''
+
+    fail = 'False'
+    
+    with open(filename) as infile:
+        lineCount = 0
+        for line in infile:
+            count = line.count("|")
+            if count != 203:
+                print("Bad column count on line " + str(lineCount))
+                fail = 'True'
+            doubleSpaceCount = line.count("  ")
+            if doubleSpaceCount > 0:
+                print("Multiple spaces on line " + str(lineCount))
+                fail = 'True'
+            lineCount = lineCount + 1
+            
+    if fail == 'True':
```

Comment:
I'm not sure what shape this might eventually take, but if `fail` were reimagined as a boolean, the function might be a _little_ nicer if we simply `return fail` here instead. Of course, that would require updates to calling/validating this outside the test. Hmm.

Reply:
Imagine this might eventually be used in the synthea automation, but I dont really have a vision for how that'll look yet. Can always update this to better fit that solution when we get there; for now its just some manual scripting to check the manually generated files, which a printout will fit fine.